nms/app/packages/magmalte/app/components/DateTimeMetricChart.js

```diff
@@ -42,14 +42,7 @@ const useStyles = makeStyles(_ => ({
   },
 }));
 
-const CHART_COLORS = [
-  colors.secondary.dodgerBlue,
-  colors.data.flamePea,
-  'green',
-  'yellow',
-  'purple',
-  'black',
-];
+const CHART_COLORS = [colors.secondary.dodgerBlue, colors.data.flamePea];
```

Comment:
What happens if a query returns more than 2 time series?

Reply:
So far we don't have that usecase yet. Hence I just set it to two regular colors.